WORKSPACE

```diff
@@ -22,14 +22,10 @@ http_archive(
 
 http_archive(
     name = "bazel_toolchains",
-    sha256 = "1caf8584434d3e31be674067996be787cfa511fda2a0f05811131b588886477f",
-    strip_prefix = "bazel-toolchains-3.7.2",
+    sha256 = "02e4f3744f1ce3f6e711e261fd322916ddd18cccd38026352f7a4c0351dbda19",
+    strip_prefix = "bazel-toolchains-5.1.2",
     urls = [
-        "https://github.com/bazelbuild/bazel-toolchains/releases/download/3.7.2/bazel-toolchains-3.7.2.tar.gz",
-        "https://mirror.bazel.build/github.com/bazelbuild/bazel-toolchains/archive/3.7.2.tar.gz",
+        "https://mirror.bazel.build/github.com/bazelbuild/bazel-toolchains/archive/refs/tags/v5.1.2.tar.gz",
+        "https://github.com/bazelbuild/bazel-toolchains/archive/refs/tags/v5.1.2.tar.gz",
     ],
 )
-
-load("@bazel_toolchains//rules:rbe_repo.bzl", "rbe_autoconfig")
-
-rbe_autoconfig(name = "buildkite_config")
```

Comment:
Removing this, causes failure on CI:
```
(01:12:40) ERROR: /workdir/proto/private/BUILD:9:12: While resolving toolchains for target //proto/private:native: com.google.devtools.build.lib.packages.BuildFileNotFoundException: no such package '@buildkite_config//config': The repository '@buildkite_config' could not be resolved: Repository '@buildkite_config' is not defined
```

Maybe split the PR into two? One fixing toolchains other one fixing RBE?

Reply:
Yes, `rbe_autoconfig` is deprecated in `5.1.2` but we still need the config for CI. I'm looking into how to do that and will update the PR accordingly. Thanks!

Reply:
It turns out other than the usage of `rbe_autoconfig`, there was no other usage for `bazel-toolchains` so I removed it.